apps/drupal-default/particle_theme/templates/block/block--utc-image-hover-effect.html.twig

```diff
@@ -0,0 +1,106 @@
+{% extends "block.html.twig" %}
+{#
+/**
+ *
+ * Implements a grid of cards using Tailwinds layout.
+ *
+ * Available variables:
+ */
+ *
+ * This template pulls in the variables to add to hover image effects. 
+ * References the block--utc-image-hover-effect-base.html.twig to create the individual cards in this grid.
+#}
+
+{% block content %}
+	{% set rendered_content = content|render %}
+	{# Sets variable to trigger content render array. #}
+
+
+	{# I belive all of these variables defined by Drupal inputs would remain here, but the
+  grid portion would be moved to a different template file if/when a design system is implemented #}
+
+	{% set hover_count = content["#block_content"].field_hover_image_count.0.value %}
+	{% set hover_orientation = content["#block_content"].field_hover_image_orientation.0.value %}
+	
+	{% if hover_count == 1 %}
```

Comment:
what if count == 0? are we enforcing 1 as the default?

Reply:
I did think of making all the "one" fields required. Picking "none" shouldn't do anything so the person applying that would hopefully figure that out. If I implement reuseable fields for each image, then I can't make the fields required.